### PR TITLE
Added information re multiple and single availability zones.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
New PR to replace old PR that had a branch error. This content incorporates the feedback from peer reviewer and SME.

**Note:** The old PR # was https://github.com/openshift/managed-openshift-docs/pull/12, just for cross-reference purposes.

Again, no preview can be produced at this time - reviews must be done by looking at the PR.

JIRA: https://issues.redhat.com/browse/OSDOCS-1764

@yufchang   @vikram-redhat   @jeremyeder   @bergerhoffer

### CODE REVIEW

File: modules/rosa-creating-cluster.adoc [hunk 1]
```diff
@@ -15,20 +15,22 @@ You have completed the installation prerequisites.
...
++
+[source, terminal]
+----
+$ rosa create cluster --cluster-name=rh-rosa-test-cluster1
+----
++
 [NOTE]
 ====
+Multi-availability zones (AZ) are recommended for ROSA production workloads. The default is a single availability zone. Use either `--help` for an example of how to set this option or use interactive mode to be prompted for this setting.
+
```
Comment (jeremyeder) [inline modules/rosa-creating-cluster.adoc:28]: The feedback from myself and @bergerhoffer didn't make it to this new PR

Comment (jonquilwilliams) [reply modules/rosa-creating-cluster.adoc:28]: What I meant was I incorporated your feedback into the content itself, but I did not copy over either of your actual comments. But here they are for your reference:

bergerhoffer: 

I think that "either" is throwing me off on the second sentence. I think you could just say "Use --help to set the option manually or use interactive mode to be prompted for this setting."

Also, I can't imagine that the --help flag actually sets the option manually. I would think that help might just give you an example of how to set the option manually. Should it be "Use --help for an example of how to set the option manually or use interactive mode to be prompted for this setting." ?

jeder: 

Agreed to wqhat @bergerhoffer said. Also s/Multi-availability/Multiple availability zones

and s/ROSA//


I also just updated the `describe cluster` example in step 2 with the one you gave me in your separate PR this morning..

**Comment by jonquilwilliams** [comment]
@jeremyeder @bergerhoffer  This is now QE verified. If it looks good to both of you, please comment here so that Andrea can merge this to the main branch. Thanks.

**Comment by jeremyeder** [comment]
LGTM

**Comment by jonquilwilliams** [comment]
@bergerhoffer This has been QE verified in JIRA and SME verified in this PR. Please merge the branch. You may want to check with Vikram about merging details. Thanks!

**Review by bergerhoffer — COMMENTED** [review]
This LGTM now!

Since this is against a specific version branch I'm assuming that it doesn't need cherry picked anywhere else, but let me know if that is not the case.

**Pull request merged**